### PR TITLE
Fix the path of seed_nodes configuration

### PR DESCRIPTION
**Pull request by asdf2014**
image

### CODE REVIEW

**Review by neuyilan — CHANGES_REQUESTED** [review]
now, the seed nodes contain the _client_port_ , looks like the following, please correct it together.

`{IP/DOMAIN}:internal_meta_port:internal_data_port:client_port`

**Comment by asdf2014** [comment]
@neuyilan Thanks. It's done.

File: docs/UserGuide/Server/Cluster Setup.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ The configuration items described below are in the `iotdb-cluster.properties` fi
...
 |Type|String|
 |Default|127.0.0.1:9003:40010,127.0.0.1:9005:40012,127.0.0.1:9007:40014|
```
Comment (neuyilan) [inline docs/UserGuide/Server/Cluster Setup.md:103]: the default value should be change together
```suggestion
|Default|127.0.0.1:9003:40010:55560,127.0.0.1:9005:40012:55561,127.0.0.1:9007:40014:55562|
```

File: docs/zh/UserGuide/Server/Cluster Setup.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ or
 
 |名字|seed\_nodes|
 |:---:|:---|
-|描述|集群中节点的地址，`{IP/DOMAIN}:internal\_meta\_port:internal\_data\_port`格式，用逗号分割；对于伪分布式模式，可以都填写`localhost`，或是`127.0.0.1` 或是混合填写，但是不能够出现真实的ip地址；对于分布式模式，支持填写real ip 或是hostname，但是不能够出现`localhost`或是`127.0.0.1`。当使用`start-node.sh(.bat)`启动节点时，此配置意味着形成初始群集的节点，每个节点的`seed_nodes`应该一致，否则群集将初始化失败；当使用`add-node.sh(.bat)`添加节点到集群中时，此配置项可以是集群中已经存在的任何节点，不需要是用`start-node.sh(bat)`构建初始集群的节点。|
+|描述|集群中节点的地址，`{IP/DOMAIN}:internal_meta_port:internal_data_port:client_port`格式，用逗号分割；对于伪分布式模式，可以都填写`localhost`，或是`127.0.0.1` 或是混合填写，但是不能够出现真实的ip地址；对于分布式模式，支持填写real ip 或是hostname，但是不能够出现`localhost`或是`127.0.0.1`。当使用`start-node.sh(.bat)`启动节点时，此配置意味着形成初始群集的节点，每个节点的`seed_nodes`应该一致，否则群集将初始化失败；当使用`add-node.sh(.bat)`添加节点到集群中时，此配置项可以是集群中已经存在的任何节点，不需要是用`start-node.sh(bat)`构建初始集群的节点。|
 |类型|String|
 |默认值|127.0.0.1:9003:40010,127.0.0.1:9005:40012,127.0.0.1:9007:40014|
```
Comment (neuyilan) [inline docs/zh/UserGuide/Server/Cluster Setup.md:95]: same as above

**Review by neuyilan — CHANGES_REQUESTED** [review]

**Review by neuyilan — APPROVED** [review]

**Review by jt2594838 — APPROVED** [review]

**Comment by qiaojialin** [comment]
Hi, please fix the conflict

**Comment by asdf2014** [comment]
Hi, @qiaojialin . Thanks for your comment. It's done.

**Review by HTHou — APPROVED** [review]

**Pull request merged**